charts/rekor/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: Part of the sigstore project, Rekor is a timestamping server and tr
 
 type: application
 
-version: 0.2.15
+version: 0.2.16
```

Comment:
@k4leung4 Can the version be bumped on the scaffold chart with this change?

Reply:
done, thanks for pointing that out.